Update booktabs.sty to the version in CTAN 1.61803
quasi trivial change
---
 texmf-dist/tex/latex/booktabs/booktabs.sty |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

Index: texlive-base-2007/texmf-dist/tex/latex/booktabs/booktabs.sty
===================================================================
--- texlive-base-2007.orig/texmf-dist/tex/latex/booktabs/booktabs.sty	2008-01-16 08:07:53.000000000 +0100
+++ texlive-base-2007/texmf-dist/tex/latex/booktabs/booktabs.sty	2008-01-16 08:08:20.000000000 +0100
@@ -8,7 +8,7 @@
 %% 
 %% -----------------------------------------------------------------
 %% Author:     Simon Fear
-%% Maintainer:
+%% Maintainer: Danie Els (dnjels@sun.ac.za)
 %% 
 %% This file is part of the booktabs package for publication
 %% quality tables for LaTeX
@@ -32,7 +32,7 @@
 %% -----------------------------------------------------------------
 \NeedsTeXFormat{LaTeX2e}[1999/12/01]
 \ProvidesPackage{booktabs}
-    [2005/03/16 v1.6180 publication quality tables]
+    [2005/04/14 v1.61803 publication quality tables]
 \newdimen\heavyrulewidth
 \newdimen\lightrulewidth
 \newdimen\cmidrulewidth
@@ -69,7 +69,7 @@
 \def\@BTfnslthree{\afterassignment\@BTfnslone\let\next= }
 \def\toprule{\noalign{\ifnum0=`}\fi
   \@aboverulesep=\abovetopsep
-  \global\@belowrulesep=\aboverulesep %global cos for use in the next noalign
+  \global\@belowrulesep=\belowrulesep %global cos for use in the next noalign
   \global\@thisruleclass=\@ne
   \@ifnextchar[{\@BTrule}{\@BTrule[\heavyrulewidth]}}
 \def\midrule{\noalign{\ifnum0=`}\fi
